log/log.go

```diff
@@ -144,7 +144,11 @@ func (m *defaultLogger) PrintBitriseStartedEvent(plan models.WorkflowRunPlan) {
 			EventType: "bitrise_started",
 		})
 	} else {
-		m.PrintBitriseASCIIArt()
+		v := strings.TrimSpace(plan.Version)
+		if v == "" {
+			v = version.VERSION
+		}
```

Comment:
I think we can drop this fallback from here because it is not the logger's responsibility to make sure the version has a non-empty value. It should just log based on the input it has received.

Reply:
In that case do we need `strings.TrimSpace` from line 147?

Reply:
Yes, you are right. That can be removed as well.